packages/docs/developer-resources/celo-dapp-gallery.md

```diff
@@ -2,6 +2,12 @@
 
 Welcome to the DApp Gallery! Here are some featured examples to help you get started building on Celo.
 
+## Celo Toolkit
+
+Celo Toolkit is the open source interface for managing Celo accounts on Web. Start with Celo now in 1 minutes. Phone number or app downloads are not necessary.
```

Comment:
I think it would be helpful to make it more clear on the page that this tool kit is for the TESTNET only. This is not a tool for managing accounts on the mainnet.